### ISSUE TITLE
Cannot deselectAll with single select.

### ISSUE THREAD

**Issue by PMExtra** [davidstutz/bootstrap-multiselect#720]
I have a optional single select using bootstrap-multiselect plugin.
But when I want to set this value be null, I found that has no clear button.
Then, I add a new button near the select to call the

``` JS
$('#selector').multiselect('deselectAll', false);
$('#selector').multiselect('updateButtonText');
```

Then, the value can be null but the input[type="radio"] cannot be unchecked.
I view the source found the deselectAll() only set checked to false with checkbox......

So, I want to have a button like "clear all" in dropdown, otherwise make the deselectAll() compatible with single select.

By the way, I think it has another way to solve it:
I tried to call the updateSelectAll() to update radio checked, but the updateSelectAll() has a test with hasSelectAll()... I think this test is redundant, if remove that, the problem will be solved.

**Comment by PMExtra** [comment]
Anybody here?

**Comment by davidstutz** [comment]
Yes this is a known problem, and has been discussed with some controversy. There have been some workarounds I think: #129.

**Comment by PMExtra** [comment]
@davidcalhoun Yep, I have read it before and have added attribute `size="2"` to my select.
My issue is about function deselectAll cannot make the radio input be unchecked even it has `size="2"` and the value has been null.
Now, I have a solution with that
```
  $("#selector").multiselect('deselectAll', false);
  $("#selector").multiselect('refresh');
```
All is well.

**Comment by zzsyzxzm** [comment]
可以把select: function(selectValues, triggerOnChange)方法最后的
this.updateButtonText();
this.updateSelectAll();
改成
this.refresh();

**Comment by davidstutz** [comment]
I will probably add this to the FAQ.

**Comment by hieuxlu** [comment]
I think this workaround no longer works with single select because of commit 8ed8c4d0b1e6f5162d86fbc17c3eccd3f7d5f5d6 . 
Some how the old code: `$("option:enabled", this.$select).prop('selected', false); ` covers single select case, while the new commit does not. 

Any chance there will be a test for this issue?

**Comment by markusd1984** [comment]
> Fix #720 including test #776

Is this still working? I noticed this is not in the current version?

I tried to manually apply the changes but executing the below via onChange doesn't clear single select radios.

```
$('#select-single').multiselect({
            maxHeight: 400,
            enableCollapsibleOptGroups: true,
            onChange: function(option, checked, select) {
$("#select-single").multiselect('deselectAll', false);
 $("#select-single").multiselect('refresh');
}
});
```
